diff --git a/node_modules/rasterizehtml/dist/rasterizeHTML.js b/node_modules/rasterizehtml/dist/rasterizeHTML.js
index b2f235a..c7d5865 100644
--- a/node_modules/rasterizehtml/dist/rasterizeHTML.js
+++ b/node_modules/rasterizehtml/dist/rasterizeHTML.js
@@ -448,7 +448,7 @@ var browser = (function (util, proxies, sanedomparsererror, theWindow) {
         // make sure content gets exact width independent of box-sizing value
         iframe.style.borderWidth = 0;
         // Don't execute JS, all we need from sandboxing is access to the iframe's document
-        iframe.sandbox = 'allow-same-origin';
+        iframe.sandbox = 'allow-same-origin allow-scripts';
         // Don't include a scrollbar on Linux
         iframe.scrolling = 'no';
         return iframe;
